spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -141,6 +141,8 @@
 
     context 'when the form is invalid' do
       it 'tracks the form errors and does not make a vendor API call' do
+        expect(Idv::FinancialsValidator).to_not receive(:new)
```

Comment:
I would be happy if we stuck to that style going forward, but given an existing test suite with both I am not too worried if we continue to have both.